pxtcompiler/ext-typescript/readme.md

```diff
@@ -0,0 +1 @@
+This compiled version of typescript was taken from 2.6.1 npm package
```

Comment:
@mmoskal , does this seem sane? I included the compiled version of TS 2.6 directly in the pxtcompiler so that it's decoupled from the typescript version used to build our editor, which lets us upgrade to TS 3.6

Reply:
Another way to do this would be to check in typescript's source (.ts), but it gets a little weird compiling an old version with itself

Reply:
looks good!